app/react-transition/Link.spec.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import Link from "./Link";
+
+test("Link accepts a route string, replace bool, id, className, title, and children", () => {
+  const component = renderer.create(
```

Comment:
We probably need to set up a fake angular app before this. So that `angular.element(document)` returns something useful